core/dbt/artifacts/resources/v1/config.py

```diff
@@ -122,6 +124,7 @@ class NodeConfig(NodeAndTestConfig):
         default_factory=ContractConfig,
         metadata=MergeBehavior.Update.meta(),
     )
+    event_time: Any = None
```

Comment:
Adding the event_time, batch_size, and lookback fields as `Any` to avoid any regressions (e.g. parsing failures for projects that make use of these configs). 

We will be adding parse-time validation for microbatch models in a downstream PR.